### PR TITLE
Use execroot for compiler working directory

### PR DESCRIPTION
**Pull request by LeFrosch**
Changes the working directory of the compiler to the current execroot. Also adds an integration test for the custom llvm toolchain.

### CODE REVIEW

File: base/src/com/google/idea/blaze/base/scope/BlazeContext.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import com.google.idea.blaze.common.Output;
 import com.google.idea.blaze.common.PrintOutput;
 import com.google.idea.blaze.exception.BuildException;
+import com.intellij.openapi.application.ApplicationManager;
```
Comment (ujohnny) [inline base/src/com/google/idea/blaze/base/scope/BlazeContext.java:29]: looks unrelated (and the one below)

File: clwb/tests/projects/llvm_toolchain/WORKSPACE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```
Comment (ujohnny) [inline clwb/tests/projects/llvm_toolchain/WORKSPACE:1]: let's move to bzlmod for this test because we use bazel 7+ anyway

**Review by ujohnny — CHANGES_REQUESTED** [review]

File: clwb/tests/projects/llvm_toolchain/WORKSPACE (hunk 2 above)
Comment (LeFrosch) [reply clwb/tests/projects/llvm_toolchain/WORKSPACE]: I migrated the test, but I cannot delete the WORKSPACE file because `rules_bazel_integration_test` requires a WORKSPACE file.
